### DISCUSSION TITLE
[Enhancement] [input-number] el-input-number的输入框展示问题

### DISCUSSION THREAD

**Discussion by PeyFon** [element-plus/element-plus#9453] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `input-number`

## Description

el-input-number当precision设置为0时，虽然实际存下来的数据为整数，但给用户看的时候仍旧带小数点 如： 1.  相似的问题还有例如：当我精度为2时，我仍旧可以输入2.1234 它会在失去焦点时才处理成2位精度。  我认为处理精度的时机最好不要在blur失去焦点时处理，应该在input输入时就处理
图例：
![image](https://user-images.githubusercontent.com/60815493/186564711-b1c3769b-0d09-4080-a7c7-5cd89fbfb02b.png)

### Additional comments

https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtaW5wdXQtbnVtYmVyIHYtbW9kZWw9XCJudW1cIiA6bWluPVwiMVwiIDptYXg9XCIxMFwiIEBjaGFuZ2U9XCJoYW5kbGVDaGFuZ2VcIiA6Y29udHJvbHM9J2ZhbHNlJyA6cHJlY2lzaW9uPScwJyAvPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgbnVtID0gcmVmKDEpXG5jb25zdCBoYW5kbGVDaGFuZ2UgPSAodmFsdWU6IG51bWJlcikgPT4ge1xuICBjb25zb2xlLmxvZyh2YWx1ZSlcbn1cbjwvc2NyaXB0PlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319

**Comment by ldc2726** [comment]
而且我不知道你发现了没有，内容为1的时候不能输入点，而其他数字可以。
